### PR TITLE
Finalised 0.3.x Release

### PR DESCRIPTION
**Pull request by hjabbot**
# PolarRoute Pull Request Template

Date: 2023-10-28
Version Number: 0.3.7  
 
## Description of change
Finalising next major release of PolarRoute

# Testing
To ensure that the functionality of the PolarRoute codebase remains consistent throughout the development cycle a testing strategy has been developed, which can be viewed in the document `test/testing_strategy.md`. 
This includes a collection of test files which should be run according to which part of the codebase has been altered in a pull request. Please consult the testing strategy to determine which tests need to be run. 

- [ ] My changes have not altered any of the files listed in the testing strategy

- [ ] My changes result in all required regression tests passing without the need to update test files.  

- [x] My changes require one or more test files to be updated for all regression tests to pass.   
> Changed in prior PR's, won't reiterate here
>  All tests passed, but for some reason I can't upload the output txt file. ~64000 warnings which seems normal

# Checklist

- [x] My code follows [pep8](https://peps.python.org/pep-0008/) style guidelines.  
- [x] I have commented my code, particularly in hard-to-understand areas.  
- [x] I have updated the documentation of the codebase where required.  
- [x] My changes generate no new warnings.   
- [ ] My PR has been made to the `0.3.x` branch (**DO NOT SUBMIT A PR TO MAIN**)

### CODE REVIEW

**Review by Ulvetanna — APPROVED** [review]
All seems good, subject to another reviewer doing a detailed review of the manual pages through the .html forms. @gecoombs can you complete this;

**Review by gecoombs — APPROVED** [review]
I've made a few additional change to the docs and regenerated the html files. Otherwise this looks good to me :+1:

File: docs/html/_sources/index.rst.txt
Comment (ayat-khairy) [inline docs/html/_sources/index.rst.txt:1]: I think PolarRoute leverages MeshiPhi to have a digital ocean representation. But PolarRoute is not an extension of MeshPhi as they both serve different purposes.

**Review by ayat-khairy — APPROVED** [review]

**Review by SamuelHall700 — APPROVED** [review]
Looks good, happy to approve the merge

**Comment by gecoombs** [comment]
MeshiPhi is now available on PyPI so this release is ready to merge :slightly_smiling_face: : https://pypi.org/project/meshiphi/

**Pull request merged**